templates-assets/templates/NotificationAAR_RADDalt/i18n/de.json

```diff
@@ -32,5 +33,6 @@
   "delegate_list_2":"ihr Ausweisdokument;",
   "delegate_list_3":"ihre Steuernummer;",
   "delegate_list_4":"eine Kopie oder das Original der Steuernummer und des Ausweisdokuments des Bevollmächtigenden",
-  "delegate_list_5":"un documento che attesti il tuo ruolo di legale rappresentante."
+  "delegate_list_5":"un documento che attesti il tuo ruolo di legale rappresentante.",
```

Comment:
here the sentence is "un documento che attesti il ruolo di legale rappresentante del delegante."